docs/getting_started/install_ez_platform.md

```diff
@@ -127,6 +127,34 @@ run the following command:
     composer create-project ibexa/commerce-skeleton .
     ```
 
+??? note "Using PHP 8"
+
+    If you are using PHP 8, use a different set of commands:
+
+    === "[[= product_name_content =]]"
+
+        ``` bash
```

Comment:
The previous part says:
```
To use Composer to instantly create a project in the current folder with all the dependencies,
run the following command:
composer create-project ibexa/content-skeleton .
```
So these examples should look like this:
```
composer create-project ibexa/content-skeleton . --no-install
composer update
```

to make things consistent